### ISSUE TITLE
Need to tune performance for MXNet & TensorFlow for seq2seq

### ISSUE THREAD

**Issue by astonzhang** [d2l-ai/d2l-en#2099]
http://preview.d2l.ai.s3-website-us-west-2.amazonaws.com/d2l-en/master/chapter_recurrent-modern/seq2seq.html
http://preview.d2l.ai.s3-website-us-west-2.amazonaws.com/d2l-en/master/chapter_attention-mechanisms/bahdanau-attention.html
http://preview.d2l.ai.s3-website-us-west-2.amazonaws.com/d2l-en/master/chapter_attention-mechanisms/transformer.html

We need to tune performance for MXNet & TensorFlow to obtain similar performance of PyTorch for each section, such as learning rate & max_epochs.

**Comment by AnirudhDagar** [comment]
http://preview.d2l.ai.s3-website-us-west-2.amazonaws.com/d2l-en/master/chapter_attention-mechanisms/bahdanau-attention.html was fixed in #2104. seq2seq and transformers for TF still need fixing.
